web/src/main/resources/application.properties

```diff
@@ -1,13 +1,9 @@
 #spring.mvc.format.date=yyyy-MM-dd
 
-spring.datasource.url=jdbc:mysql://localhost:3306/meet2gether?createDatabaseIfNotExist=true&serverTimezone=UTC&useLegacyDatetimeCode=false
-spring.datasource.username=meet2getherUser
-spring.datasource.password=superSecretPassword
-
-! W celu uruchomienia aplikacji w kontenerze dockerowym razem z baza, za pomoca docker-compose, nalezy uzyc ponizszej konfiguracji (a powyzsza zakomentowac):
-#spring.datasource.url=${SPRING_DATASOURCE_URL}
-#spring.datasource.username=${SPRING_DATASOURCE_USERNAME}
-#spring.datasource.password=${SPRING_DATASOURCE_PASSWORD}
+spring.datasource.url=${SPRING_DATASOURCE_URL:jdbc:mysql://localhost:3306/meet2gether?createDatabaseIfNotExist=true&serverTimezone=UTC&useLegacyDatetimeCode=false}
```

Comment:
Komentowanie i odkomentowywanie jest uciążliwe, mało praktyczne.
Jest kilka podejść do tematu. Tu proponuje zczytywanie wartości ze zmiennych środowiskowych i ustawianie wartości domyślnych w przypadku kiedy te zmienne nie są ustawione.

`spring.some.property=${SOME_ENV_VAR:wartosc_domyslna}`

Pierwszy `:` rozgranicza wartosc zmiennej srodowiskowej od wartosci domyslnej. Wartosc domyslna moze miec wiecej dwukropkow, to nie przeszkadza.

Reply:
Innym podejsciem (polecam) jest dzialanie tylko i wylacznie na zmiennych srodowiskowych.

`spring.some.property=${SOME_ENV_VAR}`

Zmienne srodowiskowe mozecie okreslic w intelliJ.
Podpowiem jak, jezeli jestescie zainteresowani - dajcie znac.

Dzieki temu, nie bedziecie mieli przypadkow, ze gdzies sie Wam cos odpali z wartoscia domyslna - np. z docker-compose z baza na localhost. (czy na produkcji ze zmiennymi localhost).